src/frontend/apps/drive/package.json

```diff
@@ -10,7 +10,7 @@
     "build-theme": "cunningham -g css,scss -o src/styles && mv src/styles/cunningham-tokens.scss src/styles/cunningham-tokens-sass.scss"
   },
   "dependencies": {
-    "@gouvfr-lasuite/ui-kit": "0.4.0",
+    "@gouvfr-lasuite/ui-kit": "0.4.1",
```

Comment:
We don't get the warning popup of right updating when draging from center to tree with different workspace

https://github.com/user-attachments/assets/354c24eb-5fd9-4771-94b1-581c9f122690